deployment/conf/.templates/deployment.cfg.j2

```diff
@@ -0,0 +1,37 @@
+[Workspace]
+basedir={{ basedir|default('workspace') }}
+mongodb-database={{ mongodb_database|default('workspace') }}
+mongodb-pwd={{ mongodb_pwd|default('') }}
+ws-admin={{ ws_admin|default('scanonadmin') }}
+# backend user is wstest globus user in ci
+# use that acct to generate token
+backend-token={{ backend_token|default('') }}
+mongodb-host={{ mongodb_host|default('ci-mongo') }}
+mongodb-user={{ mongodb_user|default('wsserv') }}
+service-port={{ service_port|default('7058') }}
+max-memory={{ max_memory|default('15000') }}
+min-memory={{ min_memory|default('10000') }}
+temp-dir={{ temp_dir|default('ws_temp_dir') }}
+server-threads={{ server_threads|default('20') }}
+mongodb-retry={{ mongodb_retry|default('0') }}
+ignore-handle-service={{ ignore_handle_service|default("")}}
+handle-service-url={{ handle_service|default("https://ci.kbase.us/services/handleservice/") }}
+handle-manager-url={{ handle_manager|default("https://ci.kbase.us/services/handlemngr/") }}
+# The KBase authorization server url.
+auth-service-url={{ auth_service|default("https://ci.kbase.us/services/auth/api/legacy/KBase") }}
+auth-service-url-allow-insecure={{ auth_service_url_allow_insecure|default("") }}
+auth-server-url={{ auth_server_url|default("https://ci.kbase.us/services/auth/api/legacy/KBase") }}
+# The Globus v1 authorization API url.  auth2 supports this directly
+globus-url = {{ globus_url|default("https://ci.kbase.us/services/auth/api/legacy/globus") }}
+#the credentials used for the handle manager.
+handle-manager-token={{ handle_manager_token|default("") }}
+kbase-admin-token={{ kbase_admin_token|default("") }}
+
+# configuration options for RESKE
+listeners=Search,KnowledgeEngine
+listener-Search-class=us.kbase.workspace.modules.RESKEPrototypeEventHandlerFactory
```

Comment:
The class was renamed, this will fail.

Reply:
This went in the other day as a hasty patch for testing when we pushed it into CI the other day (cut and pasted from the existing one). Tell me what the new class is called and I will make it the default in the template with an option to be overridden.

Reply:
https://github.com/kbase/workspace_deluxe/tree/dev-candidate/src/us/kbase/workspace/modules